testCourse/questions/aiGradingRubrics/question.html

```diff
@@ -0,0 +1,18 @@
+<pl-question-panel>
+  <p>To compute the mathematical expression:</p>
+
+  <!-- Sum of (2i^3) from i = 0 to i = 5 -->
+  $$
+  \sum_{i=0}^{5} 2i^3
+  $$
+
+  <p>You wrote the following Python code:</p>
+
+  <pl-code language="python">a = 0
+for i in range(5):
+    a += 3 * i ** 3</pl-code>
```

Comment:
Another sharp edge: I really want a `strip-whitespace="true"` attribute that will strip leading/trailing whitespace and does a [`textwrap.dedent()`](https://docs.python.org/3/library/textwrap.html#textwrap.dedent) so that I can write this in what'd feel like a more natural way to me.

Reply:
Yes! Issue?

Reply:
Done: https://github.com/PrairieLearn/PrairieLearn/issues/11417